pkg/controllers/osc/resources/operating_system_config.go

```diff
@@ -27,10 +27,10 @@ import (
 
 	"github.com/Masterminds/semver/v3"
 	"github.com/kubermatic/machine-controller/pkg/apis/cluster/v1alpha1"
-	providerconfigtypes "github.com/kubermatic/machine-controller/pkg/providerconfig/types"
```

Comment:
Please revert this and use it instead of copying duplicating the types across repos. We are already importing machine controller repo so we can use this as well. Plus, these settings aren't actually OSM related settings those are cluster api related settings.